extensions/speechlist.py

```diff
@@ -0,0 +1,237 @@
+            elif not Speechlistmodel.all(ctx.channel.id):
+                embed = discord.Embed(
+                    colour = discord.Colour.red(),
+                    title = 'Die Redeliste ist leer!'
+                )
+                await ctx.send(ctx.author.mention, embed=embed)
+            else:
+                Speechlistmodel.delete(ctx.channel.id, ctx.author.id)
+                new_list = Speechlistmodel.all(ctx.channel.id)
+                if not new_list:
+                    embed = discord.Embed(
+                        colour = discord.Colour.blue(),
+                        title = 'Die Redeliste ist jetzt leer. Jeder sollte dran gewesen sein :)'
+                    )
+                    await ctx.send(ctx.author.mention, embed=embed)
+                else:
+                    msg = buildMessage(new_list)
+                    await ctx.send(ctx.channel.mention, embed=msg)
+        
+        elif active == 'print' or active == 'p':
+            act_list = Speechlistmodel.all(ctx.channel.id)
+            if not act_list:
+                embed = discord.Embed(
+                    colour = discord.Colour.red(),
+                    title = 'Die Redeliste ist leer!'
+                )
+                await ctx.send(ctx.author.mention, embed=embed)
+            else:
+                msg = buildMessage(act_list)
+                await ctx.send(ctx.channel.mention, embed=msg)
+        
+        elif (active == 'delete' or active == 'd' or active == 'clear') and (ctx.author.roles.count( tutor ) >= 1 or ctx.author.roles.count( fsr ) >= 1):
+            act_list = Speechlistmodel.all(ctx.channel.id)
+            if not act_list:
+                embed = discord.Embed(
+                    colour = discord.Colour.red(),
+                    title = 'Die Redeliste ist leer!'
+                )
+                await ctx.send(ctx.author.mention, embed=embed)
+            else:
+                Speechlistmodel.deleteAll(ctx.channel.id)
+                embed = discord.Embed(
+                    colour = discord.Colour.blue(),
+                    title = 'Die Redeliste ist jetzt leer. Durch einen Tutor gelöscht!'
+                )
+                await ctx.send(ctx.author.mention, embed=embed)
+
+        elif (active == 'erase' or active == 'e') and (ctx.author.roles.count( tutor ) >= 1 or ctx.author.roles.count( fsr ) >= 1):
+            act_list = Speechlistmodel.all(ctx.channel.id)
+            if not act_list:
+                embed = discord.Embed(
+                    colour = discord.Colour.red(),
+                    title = 'Die Redeliste ist leer!'
+                )
+                await ctx.send(ctx.author.mention, embed=embed)
+            else:
+                if mem:
+                    user = ''
+                    members = self.bot.get_all_members()
+                    for member in members:
+                        if member.name == mem or member.nick == mem:
```

Comment:
@TitusKirch Do you think this could be problematic if 2 people have the same name? Maybe I should only allow this command with mentions. I should be able to get the id directly from that, right?

Reply:
I changed it to display_name, but the same question remains

Reply:
I don't think that this will be a big problem (besides, it is not used in every group)